en/docs/concepts/revoked-tokens.md

```diff
@@ -0,0 +1,25 @@
+
+### Methods to detect JWT token revocation
+
+- [Real-time Notifier](#real-time-notifier)
+- [Persistent Notifier](#persistent-notifier)
+
+![JWT revocation]({{base_path}}/assets/img/how-tos/jwt-revocation.png)
+
+#### Real-time Notifier
+
+When working with Real-time Notifications, WSO2 API Microgateway uses a Publisher- Subscriber model (pub-sub model) where the Security Token Service (STS) and WSO2 API Microgateways are linked using a Message Broker (MB). Whenever a revoke token request is received, the STS publishes a message to the JMS Message Broker. WSO2 API Microgateway has subscribed to the `tokenRevocation` topic, which is the JMS connection topic. When the JMS connection topic receives a message, the Message Broker propagates the message to the WSO2 API Microgateway servers. When the WSO2 API Microgateway servers receive this message, it will store the revoked tokens in-memory and treat them as revoked tokens. You can't extend the Real-time Notifier to add your own implementation.
+
+#### Persistent Notifier
+
+When using Persistent Notifications, WSO2 API Microgateway uses a persistent storage mechanism to link the Security Token Service (STS) and the WSO2 API Microgateway servers. Whenever a token revoke request is received, the STS publishes a message to the persistent storage. When a new WSO2 API Microgateway server spins up, it pulls the list of revoked tokens from the persistent storage, and stores them in the revoked jti (JWT ID) cache. The latter mentioned process only takes place once, and the state of the token is preserved. The state of the revoked token is used at restarts and when new API Microgateways join the cluster of Microgateway servers. By default, WSO2 API Microgateway can use WSO2 API Manager or [etcd](https://github.com/etcd-io/etcd) servers as its persistent storage when working with persistent notifications. However, unlike when using the real-time notifications, you can use any persistent storage and a custom implementation.
+
+For more information on how to enable and work with this feature, check [How to revoke tokens](/how-tos/security/rejecting-revoked-tokens)
```

Comment:
Shall we make it a note/info?

Reply:
This is the main link from concept to howto. So better keep it in the main structure.